frontend/src/metabase/auth/components/AuthLayout.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { Flex } from "grid-styled";
+import Card from "metabase/components/Card";
+
+import AuthScene from "../components/AuthScene";
+import LogoIcon from "metabase/components/LogoIcon";
+
+const AuthLayout = ({ children }) => (
```

Comment:
👍 nice extraction!

Reply:
Yeah I seriously can't believe it was the way it was this long.